eeps/eep-0048.md

```diff
@@ -101,32 +101,34 @@ the type specification below:
 
     {docs_v1,
      Anno :: erl_anno:anno(),
-     Language :: atom(),
+     BeamLanguage :: atom(),
      Format :: mime_type(),
-     ModuleDoc :: binary() | none | hidden,
+     ModuleDoc :: %{DocLanguage => DocString} | none | hidden,
```

Comment:
Is this Erlang term format or Elixir?  If Erlang should this not be `#{DocLanguage => DocString}`?

Reply:
Good catch. It is Elixir's. 😰